tools/build-overrides/build-overrides.sh

```diff
@@ -613,6 +613,39 @@ EOF
     fi
 }
 
+delete_cached_artifacts() {
+    local product=${1}
+    local product_default="all"
+    local products_valid=(all rhdm rhpam)
+    local query="rhpam*\|rhdm*"
+    if [ -z "${product}" ]; then
+        product="${product_default}"
+    else 
+        local product_valid="false"
+        for pv in ${products_valid[@]}; do
+            if [ "${pv}" = "${product}" ]; then
+                product_valid="true"
+                break
+            fi
+        done
+        if [ "${product_valid}" = "true" ] ; then
+            log_debug "Product: ${product}"
+        else
+            log_error "Invalid product: ${product}. Allowed: ${products_valid[*]}"
+            return 1
+        fi
+    fi
+    
+    if [ "${product}" != "all" ]; then
+        query="${product}*"
+    fi
+
+    for artifact in $(cekit-cache ls | grep -B5 "${query}" | grep yaml | awk '{ print substr($0,0,length($0)-6)}'); do 
+        log_info "Deleting artifact ${artifact} from local cache"
+        #cekit-cache rm $artifact;
```

Comment:
This line needs to be uncommented. I'm assuming it will delete both the artifact, and the yaml file (in the cache) that describes it?

Reply:
It's a leftover from my debug :cry: